apps/frontend/src/home/__tests__/Home.test.tsx

```diff
@@ -100,7 +100,7 @@ test('renders dashboard with correct links', async () => {
     '/news-and-events',
```

Comment:
need to work on the quality of frontend tests, this array is bullshit :see_no_evil: but not for now